cache/src/main/java/io/envoyproxy/controlplane/cache/ConfigWatcher.java

```diff
@@ -22,13 +22,18 @@ public interface ConfigWatcher {
    * @param responseConsumer   the response handler, used to process outgoing response messages
    * @param hasClusterChanged  Indicates if EDS should be sent immediately, even if version has not been changed.
    *                           Supported in ADS mode.
+   *
+   * @param allowIncompleteEdsUpdate indicates if incomplete EDS response should be sent when some clusters in request
```

Comment:
I think my first impression of "incomplete EDS response" is that it would mean we are just not sending a ClusterLoadAssignment for those clusters at all. In reality we are sending empty ones.

Maybe `useDefaultEmptyEdsUpdate` or something is more clear? hard to find a concise term for this.

Reply:
this is not just about comments, you suggests to change the variable `allowIncompleteEdsUpdate` ?